modules/java_api/cpp/openvino_java.hpp

```diff
@@ -163,7 +163,7 @@ extern "C"
     JNIEXPORT jlong JNICALL Java_org_intel_openvino_PrePostProcessor_GetPrePostProcessor(JNIEnv *, jobject, jlong);
     JNIEXPORT jlong JNICALL Java_org_intel_openvino_PrePostProcessor_Input(JNIEnv *, jobject, jlong);
     JNIEXPORT jlong JNICALL Java_org_intel_openvino_PrePostProcessor_Output(JNIEnv *, jobject, jlong);
-    JNIEXPORT void JNICALL Java_org_intel_openvino_PrePostProcessor_Build(JNIEnv *, jobject, jlong);
+    JNIEXPORT jlong JNICALL Java_org_intel_openvino_PrePostProcessor_Build(JNIEnv *, jobject, jlong);
```

Comment:
Please take attention to this change. According to https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/core/preprocess/pre_post_process.hpp, `build()` method should return a reference to `Model`.